src/components/NavBar/NavBar.vue

```diff
@@ -8,37 +8,37 @@
         max-width="70px"
         height="60px"
         position="left"
-      ></v-img>
+      />
       <v-toolbar-title class="text-h6 mr-6 hidden-sm-and-down">
         TradeShare
       </v-toolbar-title>
       <v-spacer />
-        <v-autocomplete
-          label="Search"
-          append-icon="mdi-magnify"
-          class="shrink mr-3"
-          :items="stocks"
-          :loading="IsLoading"
-          :item-text="item => item['symbol'] +': '+ item['name']"
-          rounded
-          single-line
-          solo
-          dense
-          light
-          hide-details
-          hide-no-data
-          clearable
-          click:clear
-          data-cy="autocomplete-search-bar"
```

Comment:
Don't remove this data-cy because otherwise tests will start failing as well as the one on line 36

Reply:
Ok good catch yeah I'm fixing the linting issue I'll add back the data-cy attributes

Reply:
Done